src/style.css

```diff
@@ -0,0 +1,24 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
```

Comment:
It would be good to not have the margin for the last star.

Reply:
Sure, I have just changed the margin for each "last child" in a row.